src/installer/installer.go

```diff
@@ -133,6 +133,11 @@ func (i *installer) InstallNode() error {
 		}
 		i.log.Info("Setting bootstrap node new role to master")
 
+	} else if i.Config.Role == string(models.HostRoleWorker) {
```

Comment:
why to wait for controller? i think you should just wait till 2 masters (not bootstrap) moved to Joined or Done

Reply:
OK